microsoft-edge/index.yml

```diff
@@ -20,9 +20,13 @@ highlightedContent:
 #           whats-new
   items:
     # Card
-    - title: "Download and Install: Microsoft Edge Channels"
+    - title: "Download Microsoft Edge (Official Stable Channel)"
```

Comment:
Do we need the word Official?

Reply:
The actual branch is called Official.  
![image](https://user-images.githubusercontent.com/54373752/77810136-25143f80-7050-11ea-9efa-ecd0efc4b668.png)